.github/workflows/deploy-to-control-plane-staging.yml

```
@@ -1,32 +1,94 @@
 
 # Controls when the workflow will run
 on:
-  # Uncomment the lines you want actions that will cause the workflow to Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [master]
-
-  # Allows you to run this workflow manually from the Actions tab
+    branches:
+      - 'main'
+      - 'master'
+      - ${{ github.vars.STAGING_APP_BRANCH }}
```

actionlint (1.7.4)

character ' ' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)